index.d.ts

```diff
@@ -11,7 +11,14 @@ export interface KeyMapOptions {
   action: KeyEventName;
 }
 
-export type KeySequence = MouseTrapKeySequence | KeyMapOptions | Array<MouseTrapKeySequence> | Array<KeyMapOptions>;
+export interface ExtendedKeyMapOptions extends KeyMapOptions {
```

Comment:
🐥 Adds new typescript definitions for new syntax and `getApplicationKeyMap` format.